src/Filter.js

```diff
@@ -38,15 +39,22 @@ class Filter extends Component {
             <button className="gdp-range-button">Find Countries</button>
           </form>
         </div>
-        {/* <div className={this.state.selected === 'Continent' ? 'filter-class' : 'hidden'}>
-          <ul>
-            {this.props.continentData.map((item) => {
-              return <li>{item.name}</li> 
-              })}
-         </ul>
-        </div> */}
+        <div className={this.state.selected === 'Continent' ? 'filter-class' : 'hidden'}>
+          <ul> 
+            {setTimeout(() => {
```

Comment:
What are we using the `setTimeout()` for? Usually this is a bit of a hack for something... Looks like we struggled here a bit based on the console.logs(). If you could articulate the problem I can try to guide you in the right direction.